compiler/parser/parser_test.go

```diff
@@ -99,6 +99,7 @@ func TestZtestZed(t *testing.T) {
 	require.NoError(t, err)
 	for _, z := range zed {
 		testZed(t, z)
+		return
```

Comment:
This means we'll only call testZed once.
```suggestion
```

Reply:
Oops, had that in there while debugging